tests/Fix64Tests.cs

```diff
@@ -428,12 +445,12 @@ public void Pow()
                     {
                         var expected = e == Fix64.Zero ? 1 : b == Fix64.Zero ? 0 : Math.Min(Math.Pow((double)b, (double)e), (double)Fix64.MaxValue);
 
-                        if (expected > (double)Fix64.MaxValue / 2)
-                            continue;
+                        // Absolute precision deteriorates with large result values, take this into account
+                        // Similarly, large exponents reduce precision, even if result is small.
+                        double maxDelta = Math.Abs((double)e) > 100000000 ? 0.5 : expected > 100000000 ? 10 : expected > 1000 ? 0.5 : 0.00001;
```

Comment:
Actually if the "real" Pow(x, y) > MaxValue then you expect exactly MaxValue with 0 delta.

Reply:
That would be nice, but actually the Pow approximation is still subject to the limitations that increase the maxDelta to 0.5 before. I.e. even if the expected value is MaxValue, the actual result may be slightly below, but inside the 0.5 delta.